test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -113,6 +113,89 @@ public static TheoryData MultiAttributeCheckBoxData
             Assert.Null(output.TagName); // Cleared
         }
 
+        [Theory]
+        [InlineData("bad")]
+        [InlineData("notbool")]
+        public void CheckBoxHandlesNonParsableStringsAsBoolsCorrectly(
+            string possibleBool)
+        {
+            // Arrange
+            var originalContent = "original content";
```

Comment:
BTW if these aren't really variables, and they're just constants, you could do `const string ...` instead. That way it's clearer that they're never modified.